### DISCUSSION TITLE
Make SQL queries auto naming and saving optional

### DISCUSSION THREAD

**Discussion by zozoheir** [supabase/supabase#17734] [Feature Requests]
Hi all,

I find the queries auto naming and saving extremely frustrating and annoying. Every little query I make for anything is saved and I always end up with dozens of queries I need to delete manually.  
I like to only save complicated/useful queries. Now my list is spammed with a bunch of random useless queries.

I ended up using an SQL editor now to avoid this headache. Please add a "Save query" button or something.

Thanks

**Comment by saltcod** [comment]
Thanks @zozoheir — fairly common request. Considering options for this one now.
